docs/01_introduction.md

```diff
@@ -1,48 +1,46 @@
+software installations while at the same time **providing a consistent and well performing
 scientific software stack** to end users of HPC systems.
 
-It serves as a **uniform interface for installing scientific software**,
-and saves valuable time (and frustration) by **automating** tedious, boring and repititive tasks.
-As such, it has grown out to be an **expert system** for software installation on HPC systems.
+It serves as a **uniform interface for installing scientific software**
+and saves valuable time (and frustration) by the automation of tedious, boring and repetitive tasks. 
 
-In addition, it can **empower scientific researchers to self-manage their software stack**,
-and it can serve as a tool that can be leveraged for **building optimised container images**.
+In addition, EasyBuild can **empower scientific researchers to self-manage their software stack**,
+and it can serve as a tool that can be leveraged for **building optimized container images**.
 
 The project has grown out to be a **platform for collaboration** with HPC sites worldwide.
 
 
 ### *Key features*
 
-EasyBuild is capable of **fully autonomously install of (scientific) software**,
+EasyBuild is capable of **fully autonomous installs of (scientific) software**,
 including making sure that all necessary dependencies are installed,
 and automatically generating environment module files.
 
 ***No*** **admin privileges are required**: it is sufficient to have write permissions
 to the preferred software installation prefix.
 
-It is [**highly configurable**](https://easybuild.readthedocs.io/en/latest/Configuration.html) via configuration files, environment variables and command line options.
+It is [**highly configurable**](https://easybuild.readthedocs.io/en/latest/Configuration.html) via configuration files, environment variables, and command line options.
 The functionality can be [**dynamically extended**](https://easybuild.readthedocs.io/en/latest/Including_additional_Python_modules.html) via plugins,
-and hooks are available to [**customize**](https://easybuild.readthedocs.io/en/latest/Hooks.html) the behaviour if required.
+and hooks are available for further on-the-fly [**customizations**](https://easybuild.readthedocs.io/en/latest/Hooks.html) if required.
 
-The installation procedure executed by EasyBuild is thoroughly [**logged**](https://easybuild.readthedocs.io/en/latest/Logfiles.html), and is fully **transparent** through a [dry run](https://easybuild.readthedocs.io/en/latest/Extended_dry_run.html) and [tracing](https://easybuild.readthedocs.io/en/latest/Tracing_progress.html) mechanism.
+The installation procedure executed by EasyBuild is thoroughly [**logged**](https://easybuild.readthedocs.io/en/latest/Logfiles.html) and easily inspected via [dry run](https://easybuild.readthedocs.io/en/latest/Extended_dry_run.html) and [tracing](https://easybuild.readthedocs.io/en/latest/Tracing_progress.html).
```

Comment:
Maybe `easy to inspect via`?

Is `transparent` too vague? I kind of like that term.

Reply:
Transparent sounds better, I have reverted the change.